appveyor.yml

```diff
@@ -13,7 +13,7 @@ environment:
 configuration:
   - Debug
   - Release
-version: 4.1.0.{build}
+version: 4.3.0.{build}
```

Comment:
I am pretty sure, we have to update it even once more. Current version is `4.4.0`. Now it will be `4.5.0`.